.github/workflows/unittests.yml

```diff
@@ -131,6 +134,143 @@ jobs:
           steps: ${{ toJson(steps) }}
           channel: '#cloud-tech'
 
+  standalone-smoketests:
+    name: Standalone Smoketests - Python ${{ matrix.python-version }} - ${{ matrix.variant }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash
+    needs: pre_job
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
```

Comment:
Yeah, eventually we will need to prune down the version matrix since we have like a gazillion of jobs (if not earlier, we will be able to do that once frozen binaries are in place)